tests/fsharp/Compiler/CodeGen/EmittedIL/FixedExpressionCodeGen.fs

```diff
@@ -0,0 +1,37 @@
+﻿namespace FSharp.Compiler.UnitTests.CodeGen.EmittedIL
+
+open FSharp.Compiler.UnitTests
+open NUnit.Framework
+
+[<TestFixture>]
+module FixedExpressionCodeGen =
+
+#if NETCOREAPP
+    [<Test>]
+    let ``Pinning a string emits string.GetPinnableReference`` () =
+        let code = """
+module FixedOnAString
+#nowarn "9"
+let foo(s : string) =
+    use ptr = fixed s
+    ptr
+"""
+        let il = """
+char& modreq([System.Runtime]System.Runtime.InteropServices.InAttribute) [System.Runtime]System.String::GetPinnableReference()
```

Comment:
We should check to make sure the local char is pinned as well from locals init.

Reply:
Yeah, thinking about it more this is a bad test. I'll revisit this one